### PR TITLE
First cut of stand-alone Python docs

### LINKED ISSUE #77: Integrate Python documentation into repo
Need to integrate the [Python Client](https://github.com/monoDriveIO/simulator-python-client) documentation into this repo so it can be published into the `readthedocs` site. Should be analogous to the LabView Client documentation.

### PR DESCRIPTION
**Pull request by bradleynull**
This implements a basic translation of the Python Client's README to
the documentation repo and gives more exposition on the currently available
examples.

Closes #77

### CODE REVIEW

File: docs/python_client/examples.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+simulator.start()
+```
+
+Here the simulator is being created from several of the provided configuration 
+files for "Closed Loop" control of the EGO vehicle. 
+
+### Subscribing to Sensors
+
+Once the client is connected to the `simulator` object, sensors can be 
+configured and a callback function connected to their output:
```
Comment (katiemacaulay) [inline docs/python_client/examples.md:31]: Once the client is connected to the `simulator` object, sensors can be 
configured and a callback function **is** connected to their output:

File: docs/python_client/examples.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    """
+    callback to process parsed camera data
+    """
+    camera_frame = frame
+    # Process the frame of camera data
+```
+
+### Sending Control Commands
+
+To send vehicle controls to the `simulator` object for moving the EGO vehicle:
```
Comment (katiemacaulay) [inline docs/python_client/examples.md:55]: Maybe add- **In order** to send vehicle controls to the `simulator` object for moving the EGO vehicle:

File: docs/python_client/examples.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+```python
+forward = 1
+right = 0
+brake = 0
+drive_mode = 1
+simulator.send_control(forward, right, brake, drive_mode)
+``` 
+
+Here the `forward`, `right`, and `brake` amount floating point numbers between
```
Comment (katiemacaulay) [inline docs/python_client/examples.md:65]: Here the `forward`, `right`, and `brake` amount floating point numbers are between

File: docs/python_client/examples.md [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+simulator.stop()
+```
+
+## Camera Annotation
+
+The example code in the `examples/camera_annotation.py` demonstrates how to 
+connect to an instance of the monoDrive Simulator and:
+
+* Initiate "Replay" mode 
+* Subscribe to a Camera sensor's output and  process the annotations in the frame provided by the simulation
```
Comment (katiemacaulay) [inline docs/python_client/examples.md:84]: Remove one space after `and`:  output and   process

File: docs/python_client/examples.md (hunk 1 above)
Comment (bradleynull) [reply docs/python_client/examples.md:31]: Fixed

File: docs/python_client/examples.md (hunk 2 above)
Comment (bradleynull) [reply docs/python_client/examples.md:55]: Fixed.

File: docs/python_client/examples.md (hunk 3 above)
Comment (bradleynull) [reply docs/python_client/examples.md:65]: Fixed to "~~amount~~ values are floating points numbers"

File: docs/python_client/examples.md (hunk 4 above)
Comment (bradleynull) [reply docs/python_client/examples.md:84]: Fixed.

**Comment by bradleynull** [comment]
@katiemacaulay Addressed all your PR comments. Ready for another look!

File: docs/python_client/quick_start.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+### Troubleshooting
+
+- If the simulator is running on another machine, you will need to update the
+host information in `examples/configuration/simulator.json`
+
+```json
+"simulator_ip": <IP OF SIMULATOR MACHINE>
+```
+
+- On running the script, you should see that the client is successfully
```
Comment (katiemacaulay) [inline docs/python_client/quick_start.md:77]: Rephrase without second person

Comment (bradleynull) [reply docs/python_client/quick_start.md]: Fixed. Found a few others too.

**Comment by bradleynull** [comment]
@katiemacaulay Ready for another look!

**Pull request merged**